Data/Graph/Dom.hs

```diff
@@ -41,18 +46,24 @@ module Data.Graph.Dom (
 ) where
 
 import Data.Monoid(Monoid(..))
-import Data.Bifunctor
 import Data.Tuple (swap)
 
 import Data.Tree
 import Data.List
 import Data.IntMap(IntMap)
 import Data.IntSet(IntSet)
-import qualified Data.IntMap.Strict as IM
 import qualified Data.IntSet as IS
 
+import Control.Applicative
 import Control.Monad
+
+#if __GLASGOW_HASKELL__ >= 800
```

Comment:
I'd rather have it that you use the Cabal-defined `MIN_VERSION_containers(0, 5, 0)` macro here.